clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -775,21 +769,21 @@ CVarSet ProgramInfo::getVariable(clang::Decl *D, clang::ASTContext *C) {
   }
 }
 
-std::set<FVConstraint *> *
-    ProgramInfo::getExtFuncDefnConstraintSet(std::string FuncName) {
+const std::set<FVConstraint *> *
+    ProgramInfo::getExtFuncDefnConstraintSet(std::string FuncName) const {
   if (ExternalFunctionFVCons.find(FuncName) != ExternalFunctionFVCons.end()) {
-    return &(ExternalFunctionFVCons[FuncName]);
+    return &(ExternalFunctionFVCons.at(FuncName));
```

Comment:
What does `.at` do?

Reply:
Maybe you need this to return it as `const` ?

Reply:
Same as `operator[]`, but it's `const`.